GameServerLib/Content/CharData.cs

```diff
@@ -10,8 +11,22 @@ namespace LeagueSandbox.GameServer.Content
     public class PassiveData : IPassiveData
     {
         public string PassiveNameStr { get; set; } = "";
-        public string PassiveLuaName { get; set; } = "";
+        public string PassiveAbilityName { get; set; } = "";
         public int[] PassiveLevels { get; set; } = { -1, -1, -1, -1, -1, -1 };
+
+        //TODO: Extend into handling several passives, when we decide on a format for that case.
+        public static string GetPassiveAbilityNameFromScriptFile(string champName)
+        {
+            var inputPassiveFile = File.ReadAllText($"..\\..\\..\\Content\\LeagueSandbox-Scripts\\Champions\\{champName}\\Passive.cs");
```

Comment:
You would rather not hard-code this to LeagueSandbox-Scripts.

Reply:
Is there any environment-variable I can point at in this case then?

Reply:
Game has the config file for the environment variables like content location, which I believe CharData contains.

Reply:
Check Package.cs as well if you haven't already.

Reply:
There is an attached _contentManager in the CharData class, but the variables for the PackagePath and Package is empty - so I have been forced to alter the ICharData interface to get the desired strings, which you can see in my next commit.

Reply:
Using _contentManager.GetAllLoadedPackages() gives me all packages loaded together with their names and paths, here's and example picture: [Code example](https://i.imgur.com/vAudcSW.png)

Reply:
Didn't load the packages, thanks :)

Reply:
Should be resolved in the latest commit now.